### DISCUSSION TITLE
Is there an easy way to list the packages and bundles in Packs?

### DISCUSSION THREAD

**Discussion by nbro** [symfony/symfony#54917] [Q&A]
I thought of using the flag `--dry-run` (i.e. `composer require symfony/serializer-pack --dry-run`), but this doesn't work e.g. when you've already installed the pack.

**Comment by Trismegiste** [comment]
### Packages listing
To list all of the available packages, you can use the show command.

`$ composer show`

More on https://getcomposer.org/doc/03-cli.md#show-info

**Reply by MatTheCat** [reply]
Note that since packs are automatically unpacked you need the `--available` flag. Plus, using [`jq`](https://jqlang.github.io/jq/) you can show dependencies only:

```console
$ composer show --available --format=json symfony/serializer-pack | jq '.["requires"]'
{
  "phpdocumentor/reflection-docblock": "*",
  "phpstan/phpdoc-parser": "*",
  "symfony/property-access": "*",
  "symfony/property-info": "*",
  "symfony/serializer": "*"
}
```

**Reply by nbro** [reply]
Yes, I knew we could use `composer show`, but using that alone is not what I needed. 

`composer show --available symfony/serializer-pack` is fine because it contains the info I need, although it contains a lot more info than I need. I don't want to install `jq` now. But could this feature then be integrated in composer? I suppose yes.
